### ISSUE TITLE
win10 cppvsdbg 调试状态图标错误

### ISSUE THREAD

**Issue by Dzlua** [microsoft/vscode-cpptools#732]
•VSCode Version: Code 1.12.2 ( 19222cd , 2017-05-10T13:20:36.315Z)
•OS Version: Windows_NT ia32 10.0.10240
•Extensions:

Extension

Author

Version

vscode-translate chun 0.1.0 
vscode-instant-markdown dbankier 1.1.0 
githistory donjayamanne 0.2.0 
python donjayamanne 0.6.4 
cpptools ms-vscode 0.11.0 
debugger-for-chrome msjsdiag 3.1.1 
comment-snippets Tobiah 0.2.0 

Steps to Reproduce:

1.c++，设置断点并中断。
2.点击F5继续运行，程序已经运行，但是调试按钮的状态仍然是暂停状态。
3.win10 64, cppvsdbg

**Comment by sean-mcmanus** [comment]
Bing translator gives: "1.c++ set breakpoints and break.
2. click F5 to continue operation, the program has been running, but the State of the Debug button is still suspended. 3.win10 64, cppvsdbg"

**Comment by pieandcakes** [comment]
@Dzlua So are you saying that when you click F5, that it still shows the play button?

**Comment by Dzlua** [comment]
@pieandcakes Yes,it is.It should be just the wrong state of the icon, because the program is already running

**Comment by pieandcakes** [comment]
@Dzlua Should I be able to reproduce this bug with any application? I can't seem to duplicate it.

**Comment by Dzlua** [comment]
@pieandcakes 
1、 chromium源码调试的时候，只是图标错误
2、肯定是跨线程了
3、其他项目貌似没有出现，可能是我自己的问题，但没精力查看更改。
4、我有截图，但是不知道如何上传在这里
Bing translator gives: 
1, Chromium source debugging, just the icon error
2, it must be a cross thread.
3, other projects appear to have not appeared, may be my own problems, but not the energy to see the changes.
4, I have screenshots, but do not know how to upload here
